### DISCUSSION TITLE
cash on deliver does not show up

### DISCUSSION THREAD

**Discussion by robinne** [woocommerce/woocommerce#51753] [Checkout Flow]
I only want to show Cash on Delivery. This has not worked on a test wordpress site. This has not worked on Docker on localhost, this has not worked on Hostinger which is wordpress friendly and very easy to configure and setup wordpress.

<img width="884" alt="WPPhoto" src="https://github.com/user-attachments/assets/5ef4d2b4-4a3c-4e71-a4d7-24226a7984ef">
<img width="841" alt="WPPhoto1" src="https://github.com/user-attachments/assets/71970836-ec33-44e8-974d-842308b1b972">
<img width="1067" alt="WPPhoto2" src="https://github.com/user-attachments/assets/fdd3436a-6d7c-4084-8151-f3a2dcbdd961">
<img width="1525" alt="WPPhoto3" src="https://github.com/user-attachments/assets/faf54751-67df-47e7-9111-de7980a7aacb">

**Comment by alexflorisca** [comment]
Are you trying to buy a virtual product by any chance? The only thing I can think of here is that the product type isn't supported by the payment method.
